db/schema.rb

```diff
@@ -667,6 +667,7 @@
     t.uuid "copy_case_id"
     t.boolean "case_cloned"
     t.boolean "separate_representation_required"
+    t.boolean "plf_court_order"
```

Comment:
This wasn't part of the migration task. Is this correct?

Reply:
Ooops - old PR must have been out-of-sync
It should drop out as part of the rebase on main